src/PrestaShopBundle/Form/Admin/Sell/Manufacturer/ManufacturerType.php

```diff
@@ -43,27 +46,26 @@
 /**
  * Defines form for manufacturer create/edit actions (Sell > Catalog > Brands & Suppliers)
  */
-class ManufacturerType extends AbstractType
+class ManufacturerType extends TranslatorAwareType
 {
-    /**
-     * @var TranslatorInterface
-     */
-    private $translator;
-
     /**
      * @var bool
      */
     private $isMultistoreEnabled;
 
     /**
      * @param TranslatorInterface $translator
+     * @param array $locales
      * @param bool $isMultistoreEnabled
      */
     public function __construct(
         TranslatorInterface $translator,
+        array $locales,
```

Comment:
Since you decided to use `TranslatorAwareType` it forces to configure `$locales` as well (even when you don't need it). In addition, you have to configure `parent::__construct()`. Is this the way we do it now?

Reply:
`$locales` is needed by the `TranslateType` :
```
->add('description', TranslateType::class, [
                'type' => FormattedTextareaType::class,
                'locales' => $this->locales,
                'hideTabs' => false
```

Reply:
Yes, in this case, but not every time.